public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -300,7 +324,13 @@ export const LdapSettingsPage = () => {
                 </Box>
               </section>
             )}
-            {isDrawerOpen && <LdapDrawerComponent onClose={() => setIsDrawerOpen(false)} />}
+            {isDrawerOpen && (
+              <LdapDrawerComponent
+                onClose={() => setIsDrawerOpen(false)}
+                mapKeyCertConfigured={mapKeyCertConfigured}
+                setMapKeyCertConfigured={setMapKeyCertConfigured}
```

Comment:
Why don't you move these 2 to the drawer component instead of passing them from here. I don't see them being used in `LdapSettingsPage`.

Reply:
They need to be kept in the parent component in order to initilize and maintain the state of the configured fields, otherwise the state will be lost when the drawer is closed.